tests/test_geoprocessing.py

```diff
@@ -5682,6 +5682,31 @@ def test_get_raster_info_overviews(self):
         self.assertEqual(
             raster_info['overviews'], [(500, 1000), (250, 500)])
 
+    def test_get_raster_info_vsicurl(self):
+        """PGP: raster info with vsicurl."""
+        # Access test data hosted on github
+        path = '/vsicurl/https://raw.githubusercontent.com/emlys/pygeoprocessing/feature/441/tests/test_data/raster.tif'
+        raster_info = pygeoprocessing.get_raster_info(path)
+        self.assertEqual(raster_info['raster_size'], (9, 9))
+
+    def test_get_vector_info_vsicurl(self):
+        """PGP: vector info with vsicurl."""
+        # Access test data hosted on github
+        path = '/vsicurl/https://raw.githubusercontent.com/emlys/pygeoprocessing/feature/441/tests/test_data/vector.gpkg'
+        vector_info = pygeoprocessing.get_vector_info(path)
+        self.assertEqual(vector_info['bounding_box'], [-15, 14.8, -14.8, 15])
+
+    def test_raster_map_vsicurl(self):
+        """PGP: raster_map with vsicurl."""
+        # Access test data hosted on github
+        input_path = '/vsicurl/https://raw.githubusercontent.com/emlys/pygeoprocessing/feature/441/tests/test_data/raster.tif'

```

Comment:
In playing around with this, do you have a sense for where GDAL downloads the data it needs from a hosted file? Are we sure we're cleaning that up in our tests?